### PR TITLE
Link to zip folder containing template files

### PR DESCRIPTION
**Pull request by raissanorth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,14 +11,15 @@ Your folder structure should now look like this:
...
         *   css/
+        *   fonts/
         *   images/
         *   javascript/
         *   static/
         *   templates/
             *   Includes/
             *   Layout/
 
-Now we'll simply dump the all the contents of our static site (one-ring-rentals-static.zip) into the `static/` folder, preserving the file structure. Since we've used relative paths for all the assets, having the site deep into the directory structure will not break anything.
```
Comment (robbieaverill) [inline README.md:21]: Does this link not work on the site? Our markdown parser should render this as a link (in theory anyway)

Comment (raissanorth) [reply README.md:21]: image Well, it doesn't work for me (see above). There were no square brackets in the markdown, originally.

Comment (unclecheese) [reply README.md:21]: I don't think it needs a link. The idea is that you go to the `__assets/` folder and unzip the `one-ring-rentals-static.zip`. To make it more clear, it should say `__assets/one-ring-rentals-static.zip`. The "How to use this guide" on the front page explains all this, but we should be more explicit about where the file is.

**Review by unclecheese — APPROVED** [review]

**Comment by unclecheese** [comment]
Thanks for this, @raissanorth and @robbieaverill

**Pull request merged**